pkg/hubble/parser/seven/parser_test.go

```diff
@@ -72,7 +72,7 @@ func TestDecodeL7HTTPRecord(t *testing.T) {
 		ServiceInfo:         nil,
 		DropReason:          nil,
 		HTTP: &accesslog.LogRecordHTTP{
-			Code:     404,
+			Code:     0,
```

Comment:
nitpicking: Should we leave out as effectively it will be the zero value? I find the explicit Code=0 slightly confusing but I could be missing something.

```suggestion
```

Reply:
IMO it is OK to have this here for documentation purposes even when 0 is the default value. The other thing is that this change is pulled here from https://github.com/cilium/cilium/pull/18783, so we'll manage reviews for it there.

Sorry for the confusion :-)